reana_client/validation/parameters.py

```diff
@@ -0,0 +1,307 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2021 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA client parameters validation."""
+
+
+import re
+import sys
+from collections import defaultdict
+import os
+
+
+from reana_client.config import COMMAND_DANGEROUS_OPERATIONS
+from reana_client.validation.utils import run_command
+from reana_client.printer import display_message
+
+
+def validate_parameters(workflow_type, reana_yaml):
```

Comment:
Will this implementation eventually look like the one for environments? (class based)